CourseGrab/Home/HomeTableViewController.swift

```diff
@@ -114,6 +66,42 @@ class HomeTableViewController: UITableViewController {
 
 }
 
+// MARK: - Networking
+
+extension HomeTableViewController {
+
+    private func getAllTrackedCourses() {
+        NetworkManager.shared.getAllTrackedCourses().observe { result in
+            switch result {
+            case .value(let response):
+                let available = response.data.filter { $0.status == .open }
+                if !available.isEmpty {
+                    self.tableSections.append(.available(available))
+                }
+                let awaiting = response.data.filter { $0.status != .open }
+                if !awaiting.isEmpty {
+                    self.tableSections.append(.awaiting(awaiting))
+                }
+                var newSections: IndexSet = []
+
+                if newSections.count > 0 {
```

Comment:
are we going to add more code here later on? its going to always be empty right now right

Reply:
Oops it seems I accidentally pasted like 88 after line 87 instead of right before.